exercises/parallel-letter-frequency/parallel_letter_frequency_test.go

```diff
@@ -44,3 +44,25 @@ Gave proof through the night that our flag was still there;
 O say does that star-spangled banner yet wave,
 O'er the land of the free and the home of the brave?`
 )
+
+func BenchmarkSequentialFrequency(b *testing.B) {
+	b.StopTimer()
```

Comment:
I'm not familiar with everything about the timer in benchmarking, so you may have to excuse some easy questions - what will happen if these calls to `StopTimer` and `StartTimer` are removed? Will they cause reporting to be inaccurate in some way?

I see that there are no such calls in https://github.com/exercism/xgo/blob/master/exercises/leap/leap_test.go , but there are in (for example) https://github.com/exercism/xgo/blob/master/exercises/connect/connect_test.go where we didn't want to inculde `prepare(tt.board)` in the time recorded.
It doesn't look like we have anything that applies here since the timer just gets started immediately after it gets stopped, with no intervening work in between.

Are we doing something wrong in `leap` (we should call `StopTimer` and `StartTimer` but we aren't), or are the other differences between the situations that I'm not aware of?